infer.install

```diff
@@ -0,0 +1,14 @@
+bin: [
+  "infer/bin/infer"
```

Comment:
The API of Infer is not very stable right now. So it is likely that these binaries are renamed, move of deleted. Is there a way to build this list automatically based on what is in infer/bin ?